From 8a6aaa53e49fd155988d0d098495bffbfb72a464 Mon Sep 17 00:00:00 2001
From: Eric Wild <ewild@sysmocom.de>
Date: Mon, 2 Sep 2024 10:58:46 +0200
Subject: [PATCH 1/3] apps/fft: add timezone to recording filename

As proposed by Sec, makes guessing time zones much easier when sharing
files.
---
 apps/osmocom_fft | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apps/osmocom_fft b/apps/osmocom_fft
index c48ac7e..05a6de8 100755
--- a/apps/osmocom_fft
+++ b/apps/osmocom_fft
@@ -326,7 +326,7 @@ class app_top_block(gr.top_block, Qt.QMainWindow):
         s = self._fre.text()
         s = s.replace('%S', '%e' % self.src.get_sample_rate())
         s = s.replace('%F', '%e' % self.src.get_center_freq())
-        s = s.replace('%T', datetime.datetime.now().strftime('%Y%m%d%H%M%S'))
+        s = s.replace('%T', datetime.datetime.now().astimezone().strftime('%Y%m%d%H%M%S%z'))
         return s
 
     def _set_status_msg(self, msg, timeout=0):
-- 
2.39.5

